e2e/tests/prometheus_query.rs

```diff
@@ -0,0 +1,61 @@
+use failure::{Fallible, ResultExt};
+use prometheus_query::v1::queries::{QueryData, QueryResult, VectorResult};
+use prometheus_query::v1::Client;
+#[test]
```

Comment:
I'd also add a new feature for the prom-query e2e tests, so we have easy control over which of them we want to build and run locally.

```suggestion

#[cfg(feature = "test-e2e-prom-query")]
#[test]
```

Reply:
Done